src/PatientChecking/PatientChecking/Services/ServiceModels/Patient.cs

```diff
@@ -8,19 +10,31 @@ namespace PatientChecking.Services.ServiceModels
     public partial class Patient
     {
         public Patient()
+        {
+            
+        }
+        public int PatientId { get; set; }
+        public string PatientIdentifier { get; set; } 
+        public string FullName { get; set; }
+        public DateTime DoB { get; set; }
+        public int Gender { get; set; }
```

Comment:
Please use `PatientGender` enum

Reply:
DoB's datatype has been changed to PatientGender enum